README.md

```diff
@@ -199,7 +199,7 @@ When the incoming `pendingProps` are equal to `memoizedProps`, it signals that t
 
 #### `pendingWorkPriority`
 
-A number indicating the priority of the work represented by the fiber. The [ReactPriorityLevel](https://github.com/facebook/react/blob/master/src/renderers/shared/fiber/ReactPriorityLevel.js) module lists the different priority levels and what they represent.
+A number indicating the priority of the work represented by the fiber. The [ReactFiberExpirationTime](https://github.com/facebook/react/blob/master/packages/react-reconciler/src/ReactFiberExpirationTime.new.js) module lists the different priority levels and what they represent.
```

Comment:
Broken, should be https://github.com/facebook/react/blob/main/packages/react-reconciler/src/ReactFiberLane.new.js